etc/adminhtml/system.xml

```diff
@@ -128,16 +149,14 @@
                         <depends>
                             <field id="active">1</field>
                         </depends>
-                        <config_path>reach/dhl/export_csv_hs_code</config_path>
                     </field>
-                    <field id="import_csv_hs_code" translate="label" type="Reach\Payment\Block\Adminhtml\Form\Field\Import" showInStore="1" sortOrder="130" showInDefault="1" showInWebsite="1" >
```

Comment:
This one too, will this section: ShowInStore="1" being removed mean that it won't be shown in the store config?